packages/ui/src/widgets/Preloader/Preloader.tsx

```diff
@@ -1,45 +1,86 @@
 import styled from '@emotion/styled';
+import { useMemo } from 'react';
 
 import defaultPreloaderImage from '../../assets/preloaderImage.png';
 import { Button, Stack, Typography } from '../../components';
 import { useAsyncCallback, useMediaQuery } from '../../hooks';
 
-const Image = styled.div({
+const ImageBlock = styled.div(({ hasPrelaoder }: { hasPrelaoder: boolean }) => ({
   height: 200,
   alignSelf: 'stretch',
-  backgroundImage: `url(${defaultPreloaderImage})`,
+  backgroundImage: !hasPrelaoder ? `url(${defaultPreloaderImage})` : 'none',
   backgroundSize: 'cover',
   backgroundPosition: 'center',
   borderRadius: 12,
 
+  '@media (max-height: 440px)': {
+    height: 130,
+  },
+}));
+
+const Image = styled.img({
+  height: 200,
+  width: '100%',
+  objectFit: 'fill',
+  borderRadius: 12,
   '@media (max-height: 440px)': {
     height: 130,
   },
 });
 
 const Widget = styled(Stack)({
   maxWidth: 400,
+
+  '@media (min-width: 600px)': {
+    minWidth: 400,
+  },
 });
 
 export type PreloaderProps = {
   ready?: boolean;
   onStartClick?: () => Promise<void> | void;
+  preloaderPath?: string;
+  preloaderTitle?: string;
+  preloaderDescription?: string;
 };
 
-export const Preloader = ({ ready = false, onStartClick }: PreloaderProps) => {
+export const Preloader = ({
+  ready = false,
+  onStartClick,
+  preloaderPath,
+  preloaderTitle,
+  preloaderDescription,
+}: PreloaderProps) => {
   const isLandscape = useMediaQuery('(max-height: 440px)');
   const [handleStartClick, isBusy] = useAsyncCallback(onStartClick);
 
+  const preloaderImage = useMemo(
+    () => (
+      <ImageBlock hasPrelaoder={Boolean(preloaderPath)}>
+        {Boolean(preloaderPath) && <Image src={preloaderPath} alt="" loading="lazy" />}
+      </ImageBlock>
+    ),
+    [preloaderPath],
+  );
+
+  const title = useMemo(() => (Boolean(preloaderTitle) ? preloaderTitle : 'Play now & win'), [preloaderTitle]);
```

Comment:
I don't think `useMemo` is really needed here - almost no no computation.

```suggestion
  const title = preloaderTitle || 'Play now & win';
```